### PR TITLE
Copy changes in publication form and what to do page

### PR DESCRIPTION
**Pull request by liwazi**
2 files have been amended. ready for amazon server?

### CODE REVIEW

File: node_modules/oae-avocet/publicationform/publicationform.html [hunk 1]
```diff
@@ -91,10 +91,7 @@
                     <div class="help"></div>
                 </div>
 
-                <div class="form-group">
-                    <p>__MSG__USE_THE_CAMBRIDGE_ADDENDUM_TO__ <button type="button" class="oa-trigger-modaladdendum btn oa-link oa-btn-link oa-inline-btn-link oa-btn-form-help">__MSG__FIND_OUT_MORE__</button><br/><span data-widget="addendumlink" data-label="__MSG__DOWNLOAD_THE_CAMBRIDGE_ADDENDUM__"></span>
-                    </p>
-                </div>
+
```
Comment (Coenego) [inline node_modules/oae-avocet/publicationform/publicationform.html]: These empty lines should go.

**Comment by Coenego** [comment]
Please exclude .DS_STORE files from your commit.

**Comment by Coenego** [comment]
Reassigning to @liwazi for follow-up

**Comment by liwazi** [comment]
Submit another request 0e47d83

**Comment by Coenego** [comment]
`.DS_STORE` files are still included. Can you exclude them from your commit? Back to @liwazi for follow-up.

**Comment by bp323** [comment]
I'm not sure as to why the `.DS_Store` files where included in the first place since the .gitignore file should specify them. @liwazi once they are removed I can go ahead and merge.

**Pull request closed without merging**